guides/advanced_guides/faceted_search.md

```diff
@@ -0,0 +1,192 @@
+
+- `["facetName1:facetValue1", ["facetName2:facetValue2"]]` (Array of array of strings or single strings, defaults to `null`)
+
+  Both types of array contain the facet names and values to filter on.
+  A valid array must be an array which contains either a list of strings or arrays of strings and can mix both (e.g. `["kind:t-shirt", ["color:red", "color:green"]]`).
+
+  - `facetName`: The name (the attribute) of a field used as a facet (e.g. `color`, `kind`).
+  - `facetValue`: The value of this facet to filter results on (e.g. `red`, `green`, `t-shirt`, `pants`).
+
+### Example
+
+Given a clothing dataset, suppose you want to retrieve all t-shirts with a nautical pattern. You would then use:
+
+```bash
+$ curl --get 'http://localhost:7700/indexes/clothing/search' \
+    --data-urlencode 'q=nautical' \
+    --data-urlencode 'facetFilters=["kind:t-shirt"]'
+```
+
+### Logical Connectives
+
+Inputting a double dimensional array allows you to use **logical connectives**.
+
+- **Inner arrays elements** are connected by an `OR` operator (e.g. `[["color:red", "color:green"]]`).
+- **Outer arrays elements** are connected by an `AND` operator (e.g. `["color:red", "kind:t-shirt"]`).
+
+You can mix connectives, for instance, the following array:
+
+```json
+["kind:t-shirt", ["color:red", "color:green"]]
+```
+
+Can be translated as:
+
+```SQL
+"kind:t-shirt" AND ("color:red" OR "color:green")
+```
+
+### Example
+
+Say you want to get movies matching "thriller" classified as either comedy or horror and directed by Jordan Peele.
+
+```SQL
+("genre:Horror" OR "genre:Comedy") AND "director:Jordan Peele"
+```
+
+Querying on "thriller", the above example results in the following CURL command:
+
+```bash
+$ curl --get 'http://localhost:7700/indexes/movies/search' \
+    --data-urlencode 'q=thriller' \
+    --data-urlencode 'facetFilters=[["genre:Horror", "genre:Comedy"], "director:Jordan Peele"]'
+```
+
+### 2. The facets for which to retrieve the matching count
+
+You can retrieve the count of matching terms for each facet.
+
+`facets=[<facetName>, <facetName>, ...]`
+
+This attribute can take two values:
+
+- `[<facetName>, <facetName>, ...]` (Array of strings)
+
+  An array of strings that contains the facets for which to retrieve the matching count. The number of remaining candidates for each specified facet is returned. If a facet name doesn't exist, it will be ignored.
+
+- `["*"]`
+
+  The `*` character can also be used. In that case, a count for all facets is returned.
+
+If the `facets` parameter has been set, the returned results will contain two additional fields:
+
+- `facets`: The number of remaining candidates for each specified facet.
+
+- `exhaustiveFacetsCount`:
+  Returns `true` if this count is **exhaustive**.
+  Otherwise, returns `false` if this count is **approximative**.
+
+### Example
```

Comment:
This section should be more elaborate. 
1. Add the attributes to the attributesForFacetting
2. Search example (you did this)
3. Response example